app/helpers/planning_helper.rb

```diff
@@ -0,0 +1,165 @@
+module PlanningHelper
```

Comment:
This planner helper is handling a lot of UI responsibilities, given that the project is already using view components, all the logic here should be part of a component.